packages/sights/test/build/buildJSON.test.ts

```
@@ -206,5 +214,41 @@ describe('JSON builder module', () => {
         ).toBe(true);
       });
     });
+
+    it('should property create wireframe file', () => {
+      const saveLibJSONSpy = jest.spyOn(io, 'saveLibJSON');
+      fs.existsSync = jest
+        .fn()
+        .mockImplementation((path: string) => path.includes('vehicle-key-2'));
+      buildJSONs();
+      const call = saveLibJSONSpy.mock.calls.find((args) =>
+        (args[1] as string).endsWith('wireFrames.json'),
+      );
+      expect(call?.[0]).toEqual({
+        'vehicle-key-2': {
+          [PartSelectionOrientation.FRONT_LEFT]: expect.any(String),
+          [PartSelectionOrientation.FRONT_RIGHT]: expect.any(String),
+          [PartSelectionOrientation.REAR_LEFT]: expect.any(String),
+          [PartSelectionOrientation.REAR_RIGHT]: expect.any(String),
+        },
+      } as Partial<Record<'test-key-2', PartSelectionWireFrames>>);
+    });
+
+    it('should throw error when not all the four wireframe not present', () => {
```

*should throw an error when some wireframes are missing*